executor/distsql.go

```diff
@@ -1492,7 +1492,7 @@ func (w *tableWorker) executeTask(ctx context.Context, task *lookupTableTask) er
 		chk := tryNewCacheChunk(tableReader)
 		err = Next(ctx, tableReader, chk)
 		if err != nil {
-			logutil.Logger(ctx).Error("table reader fetch next chunk failed", zap.Error(err))
+			logutil.Logger(ctx).Info("table reader fetch next chunk failed", zap.Error(err))
```

Comment:
It seems it's hard to reach this line in the unit tests.

Reply:
Now tidb is in go1.21. in Go 1.21, The new [AfterFunc](https://tip.golang.org/pkg/context/#AfterFunc) function registers a function to run after a context has been cancelled. so maybe we check it to set log level.

Reply:
WithCancelCause will be better?

Reply:
I found that sometimes this log still has unique value. It feels better to keep it as it is.